dev/release/rat_exclude_files.txt

```diff
@@ -104,6 +104,7 @@ go/*.s
 js/.npmignore
 js/closure-compiler-scripts/*
 python/cmake_modules
+python/cmake_modules/*
```

Comment:
Because it's symlinked from cpp/cmake_modules which is already checked.

Reply:
We expand symlink in tar.gz because BSD tar that is used by MSYS2 package doesn't support symlink on Windows.

Reply:
See also #2417 .

Reply:
Thanks for the clarification, good to know that!